### ISSUE TITLE
Lots of warnings on latest version

### ISSUE THREAD

**Issue by asinghvi17** [JuliaGeo/GDAL.jl#79]
Haven't been able to figure out the exact lines yet, will dig into it later.

```
WARNING: Method definition cplemergencyerror(Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:14 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:14.
┌ Warning: Replacing docs for `GDAL.cplemergencyerror :: Tuple{Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplerrorreset() in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:23 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:23.
┌ Warning: Replacing docs for `GDAL.cplerrorreset :: Tuple{}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplgetlasterrorno() in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:35 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:35.
┌ Warning: Replacing docs for `GDAL.cplgetlasterrorno :: Tuple{}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplgetlasterrortype() in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:47 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:47.
┌ Warning: Replacing docs for `GDAL.cplgetlasterrortype :: Tuple{}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplgetlasterrormsg() in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:59 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:59.
┌ Warning: Replacing docs for `GDAL.cplgetlasterrormsg :: Tuple{}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplgeterrorcounter() in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:71 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:71.
┌ Warning: Replacing docs for `GDAL.cplgeterrorcounter :: Tuple{}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplgeterrorhandleruserdata() in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:83 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:83.
┌ Warning: Replacing docs for `GDAL.cplgeterrorhandleruserdata :: Tuple{}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplerrorsetstate(Any, Any, Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:94 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:94.
┌ Warning: Replacing docs for `GDAL.cplerrorsetstate :: Tuple{Any,Any,Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplcleanuperrormutex() in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:101 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:101.
┌ Warning: Replacing docs for `GDAL.cplcleanuperrormutex :: Tuple{}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplloggingerrorhandler(Any, Any, Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:112 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:112.
┌ Warning: Replacing docs for `GDAL.cplloggingerrorhandler :: Tuple{Any,Any,Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cpldefaulterrorhandler(Any, Any, Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:123 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:123.
┌ Warning: Replacing docs for `GDAL.cpldefaulterrorhandler :: Tuple{Any,Any,Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplquieterrorhandler(Any, Any, Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:134 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:134.
┌ Warning: Replacing docs for `GDAL.cplquieterrorhandler :: Tuple{Any,Any,Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplturnfailureintowarning(Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:143 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:143.
┌ Warning: Replacing docs for `GDAL.cplturnfailureintowarning :: Tuple{Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplseterrorhandler(Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:158 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:158.
┌ Warning: Replacing docs for `GDAL.cplseterrorhandler :: Tuple{Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplseterrorhandlerex(Any, Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:175 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:175.
┌ Warning: Replacing docs for `GDAL.cplseterrorhandlerex :: Tuple{Any,Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplpusherrorhandler(Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:187 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:187.
┌ Warning: Replacing docs for `GDAL.cplpusherrorhandler :: Tuple{Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplpusherrorhandlerex(Any, Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:201 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:201.
┌ Warning: Replacing docs for `GDAL.cplpusherrorhandlerex :: Tuple{Any,Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplsetcurrenterrorhandlercatchdebug(Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:213 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:213.
┌ Warning: Replacing docs for `GDAL.cplsetcurrenterrorhandlercatchdebug :: Tuple{Any}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition cplpoperrorhandler() in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:222 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:222.
┌ Warning: Replacing docs for `GDAL.cplpoperrorhandler :: Tuple{}` in module `GDAL`
└ @ Base.Docs docs/Docs.jl:223
WARNING: Method definition _cplassert(Any, Any, Any) in module GDAL at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:233 overwritten at /builds/JuliaGPU/AbstractPlotting-jl/.julia/packages/GDAL/JO0C1/src/cpl_error.jl:233.
┌ Warning: Replacing docs for `GDAL._cplassert :: Tuple{Any,Any,Any}` in module `GDAL`
```

**Comment by visr** [comment]
Woops, fixed this in 394d7772c2e163342563e64b889897e8f0dadf1d. I'll tag a new release.

**Comment by asinghvi17** [comment]
Thanks!
